plugins/Marketplace/LicenseKey.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * Piwik - free/libre analytics platform
+ *
+ * @link http://piwik.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+namespace Piwik\Plugins\Marketplace;
+
+use Piwik\Config;
+use Piwik\Option;
+use Piwik\Plugin;
+
+class LicenseKey
+{
+    public function get()
+    {
+        return Option::get('marketplace_license_key');
```

Comment:
We'll store the license key in plain text in option table. We never expose it to UI but it is used in backend when making requests to Marketplace.